amoro-ams/src/main/resources/mysql/ams-mysql-init.sql

```diff
@@ -179,6 +179,21 @@ CREATE TABLE `task_runtime`
     KEY  `table_index` (`table_id`, `process_id`)
 ) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4 COMMENT 'Optimize task basic information';
 
+CREATE TABLE `table_process_state`
+(
+    `process_id`                    bigint(20) NOT NULL COMMENT 'optimizing_procedure UUID',
+    `action`                        varchar(10) NOT NULL COMMENT 'process action',
```

Comment:
The length of 10 maybe too small.


`    `action`                        varchar(10) NOT NULL COMMENT 'process action',`

Reply:
What do you suggest?

Reply:
I make it to 16 and make  name length checking.
long names would cause unnecessary overhead in databases